### ISSUE TITLE
Have you reached any progress?

### ISSUE THREAD

**Issue by iHold** [vmelamed/Vertica.EF#1]
Really wanted to write you but did not know where. So I wrote here. I would like to ask: have you achieved any results? The fact that I have an urgent need for such development. And I would be very grateful if you shared any experience. Thank you.

**Comment by vmelamed** [comment]
Yeah, I got pretty far ahead with this. I wanted to do a lot more testing but then things around me changed. Feel free to fork it.
You can contact me at valo@vtmelamed.com.

**Comment by vilinski** [comment]
hahaha! you are trolling us, do you? orm for vertica?
